### ISSUE TITLE
add shortcuts reset wallpaper[discussion]

### ISSUE THREAD

**Issue by zosy** [linuxmint/Cinnamon#5600]
after #5599 ,i followed #5513 ,i reinstall brand new linuxmint 18,all default settings,trying add short cut, though not reset wallpaper, but the log same generate in to .xsession-error:

(cinnamon:7148): St-CRITICAL **: st_widget_get_theme_node called on the widget [0x5652c5d42010 StScrollView.popup-sub-menu] which is not in the stage.

(cinnamon:7148): St-CRITICAL **: st_widget_get_theme_node called on the widget [0x5652c5d1c340 StScrollView.popup-sub-menu] which is not in the stage.

(cinnamon:7148): St-CRITICAL **: st_widget_get_theme_node called on the widget [0x5652c96dc830 StScrollView.popup-sub-menu] which is not in the stage.

linuxmint18 show is issue not frequency, i have tryed lot of time, when you add the 7 and 8 or 10 shortcut maybe, then click Add, interface will not responding about 2 seconds, the errors show up, although not reset the wallpaper, but i still think some wrong with it , and ENV s are differents,such as fedora 24, centos 7 and debian 8.5, cause shows reset wallpaper bug.

**Comment by JosephMcc** [comment]
We don't need another issue for this. Please just use https://github.com/linuxmint/Cinnamon/issues/5513
